lib/salus/scanners/report_node_modules.rb

```diff
@@ -103,6 +110,36 @@ def record_dependencies_from_package_lock_json
       end
     end
 
+    def record_dependencies_from_bower_json
+      packages = JSON.parse(@repository.bower_json)
+      # Record if custom registry is defined in .bowerrc
+      # Supports single and multiple registry -
+      # {"registry": "https://registry.bower.io"}
+      # {"registry": {"search": ["http://localhost:9000","https://registry.bower.io"] }}
+      sources = '<bower manager default>'
```

Comment:
Is the actual `bower manager default` url defined in the bowser file?  It would be better to include an actual url instead of `bower manager default`.